tests/sentry/tasks/test_post_process.py

```diff
@@ -375,6 +375,28 @@ def test_skipping_an_issue_doesnt_mark_it_processed(self, mock_derive_code_mappi
             self._call_post_process_group(charlie_event2)
             assert mock_derive_code_mappings.delay.call_count == 2
 
+    # XXX: Delete this test once we've migrated
```

Comment:
@markstory these two tests will be deleted in the future but it tests that the right queue is used.